apps/common/utils/index.tsx

```diff
@@ -49,14 +48,14 @@ export function getVaultRawAPY(vaults: TDict<TYDaemonVault | undefined>, vaultAd
 	return 0;
 }
 
-export function getAmountWithSlippage(from: string, to: string, value: BigNumber, slippage: number): number {
+export function getAmountWithSlippage(from: string, to: string, value: bigint, slippage: number): number {
 	const	hasLP = (toAddress(from) === LPYCRV_TOKEN_ADDRESS|| toAddress(to) === LPYCRV_TOKEN_ADDRESS);
 	const	isDirectDeposit = (toAddress(from) === YCRV_CURVE_POOL_ADDRESS || toAddress(to) === LPYCRV_TOKEN_ADDRESS);
 
 	if (hasLP && !isDirectDeposit) {
-		const minAmountStr = Number(ethers.utils.formatUnits(formatBN(value), 18));
-		const minAmountWithSlippage = ethers.utils.parseUnits((minAmountStr * (1 - (slippage / 100))).toFixed(18), 18);
-		return formatToNormalizedValue(formatBN(minAmountWithSlippage), 18);
+		const minAmountStr = Number(formatUnits(toBigInt(value), 18));
+		const minAmountWithSlippage = parseUnits(((minAmountStr * (1 - (slippage / 100))).toFixed(18) as `${number}`), 18);
```

Comment:
I think this is one of the examples where casting is acceptable, as we know it's a `${number}` for sure